Header header
robot_msgs/MechanismState mech_state
image_msgs/ImagePointStamped[] points
